src/current/v24.2/performance-recipes.md

```diff
@@ -297,6 +303,14 @@ A low percentage of live data can cause statements to scan more data ([MVCC valu
 
 Reduce the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) zone configuration of the table as much as possible.
 
+### KV DistSender batches being throttled (performance impact to larger clusters)
+
+If you see values greater than `0` for the `distsender.batches.async.throttled` metric, consider increasing the [KV layer DistSender]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#distsender) concurrency using the `kv.dist_sender.concurrency_limit` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}).
```

Comment:
Small refinements:

```
If you see `distsender.batches.async.throttled` values that aren't zero (or aren't consistently near zero), experiment with increasing the [KV layer DistSender]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#distsender) and [KV layer Streamer]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#streamer) concurrency using the `kv.streamer.concurrency_limit` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}).  In 24.3, these default values were increased by 6x and 12x, respectively.  For versions older than 24.3, increasing the value by 6x and 12x would be a good starting point.
```

Reply:
In my testing I didn't see any increase in memory, so I think the concern here is potentially overblown.  I'm sure it's a possibility, but not wasn't something I experienced or measured.  And, certainly for any cluster or workload running into these limits they're going to have enough spare RAM that a ~1-10MB of RAM will go unnoticed given the increase in throughput.